citest/gcp_testing/resource_snapshot.py

```diff
@@ -303,6 +317,142 @@ def stringify(self, resource):
     return '\n'.join(lines)
 
 
+class ApiDiff(object):
```

Comment:
Might be worth collecting the api family diffs in another class and using that to clean up the representation.